src/jukebox/components/rfid/hardware/template_new_reader/template_new_reader.py

```diff
@@ -48,7 +48,7 @@ class ReaderClass(ReaderBaseClass):
     All the required interfaces are implemented there.
 
     Put your code into these functions (see below for more information)
-      - __init__
+      - `__init__`
```

Comment:
Why is this changed?

Reply:
Otherwise the double underscores would render as italic formatting